contrib/detect_copy_paste_sa.py

```diff
@@ -50,9 +50,9 @@ def suffix_array(s, k=256, n=None):
         n = len(s)
         s += [0, 0, 0]
 
-    n0 = (n + 2) / 3
-    n1 = (n + 1) / 3
-    n2 = n / 3
+    n0 = int((n + 2) / 3)
```

Comment:
There’s an idiomatic way to do this in Python3: `(n + 2) // 3` i.e. true integer division directly. Could you use this pattern instead of the float division + reconversion to int?

Reply:
It seems reasonable, it is implemented in next commit :)

Reply:
Thanks!